packages/frontend/editor-ui/index.html

```diff
@@ -6,11 +6,7 @@
 		<meta name="viewport" content="width=device-width,initial-scale=1.0" />
 		<link rel="icon" href="/favicon.ico" />
 		<style>@media (prefers-color-scheme: dark) { body { background-color: rgb(45, 46, 46) } }</style>
-		<script type="text/javascript">
-			window.BASE_PATH = '/{{BASE_PATH}}/';
-			window.REST_ENDPOINT = '{{REST_ENDPOINT}}';
-		</script>
-		<script src="/{{REST_ENDPOINT}}/sentry.js"></script>
+		<script src="./config.js"></script>
```

Comment:
This should probably be
```suggestion
		<script src="/config.js"></script>
```

Getting an error if I open e.g. `http://localhost:5678/home/workflows`